slycot/analysis.py

```diff
@@ -1228,4 +1228,332 @@ def ab09nd(dico,job,equil,n,m,p,A,B,C,D,alpha=None,nr=None,tol1=0,tol2=0,ldwork=
+
+         low = 0   and   high = delta,
+
+    and delta is a small number approximately equal to the square root
+    of machine precision times the Frobenius norm (Euclidean norm)
+    of A. If A is stable in the sense that all eigenvalues of A lie
+    in the open left half complex plane, then beta(A) is the distance
+    to the nearest unstable complex matrix, i.e., the complex
+    stability radius.
+
+    Required arguments:
+        n : input int
+            The order of the matrix A.  n >= 0.
+        A : input rank-2 array('d') with bounds (n,n)
+            The leading n-by-n part of this array must contain the matrix A.
+
+    Optional arguments:
+        tol : Specifies the accuracy with which low and high approximate
+              beta(A). If the user sets tol to be less than sqrt(eps),
+              where eps is the machine precision (see LAPACK Library
+              Routine DLAMCH), then the tolerance is taken to be
+              sqrt(eps).
+              The recommended value is tol = 9, which gives an estimate
+              of beta(A) correct to within an order of magnitude.
+
+    Return objects:
+        low : float
+              A lower bound for beta(A).
+        high : float
+               An upper bound for beta(A).
+    """
+    out = _wrapper.ab13ed(n, A, tol)
+    if out[-1] == 0:
+        # success
+        return out[0], out[1]
+    elif out[-1] < 0:
+        hidden = ' (hidden by the wrapper)'
+        arg_list = ['n', 'A', 'lda'+hidden, 'low'+hidden, 'high'+hidden, 'tol',
+                    'dwork'+hidden, 'ldwork'+hidden, 'info'+hidden]
+        error_text = "The following argument had an illegal value: " + arg_list[-out[-1]-1]
+        e = ValueError(error_text)
+        e.info = out[-1]
+        raise e
+    elif out[-1] == 1:
+        e = ArithmeticError("the QR algorithm fails to converge")
+        e.info = out[-1]
+        raise e
+    else:
+        raise RuntimeError("unknown error code %r" % out[-1])
+
+def ab13fd(n, A, tol = 0.0):
+    """beta, omega = ab13fd(n, A, [tol])
+
+    To compute beta(A), the 2-norm distance from a real matrix A to
+    the nearest complex matrix with an eigenvalue on the imaginary
+    axis. If A is stable in the sense that all eigenvalues of A lie
+    in the open left half complex plane, then beta(A) is the complex
+    stability radius, i.e., the distance to the nearest unstable
+    complex matrix. The value of beta(A) is the minimum of the
+    smallest singular value of (A - jwI), taken over all real w.
+    The value of w corresponding to the minimum is also computed.
+
+    Required arguments:
+        n : input int
+            The order of the matrix A.  n >= 0.
+        A : input rank-2 array('d') with bounds (n,n)
+            The leading n-by-n part of this array must contain the matrix A.
+
+    Optional arguments:
+        tol : Specifies the accuracy with which beta(A) is to be
+              calculated. (See the Numerical Aspects section below.)
```

Comment:
The section Numerical Aspects, which is referenced here, was not also copied from the FORTRAN sources. Can you either write that the documentation of the FORTRAN source code should be consulted for details, or copy the Numerical Aspects section ([lines 124-143 of AB13FD.f](https://github.com/python-control/Slycot/blob/9a098d124cb31c083d26feae746a1f98e26d1a90/slycot/src/AB13FD.f#L124-L143)) into this docstring?